app/views/partials/foyer/logement.html

```diff
@@ -165,7 +158,7 @@
       </div>
     </div>
 
-    <div class="form-group" ng-if="cityStartsWith('Paris')">
+    <div class="form-group" ng-if="cityStartsWith('Paris') && captureCodePostal()">
```

Comment:
The condition seems reversed to me. Remember that JS does short-circuit evaluation. This means that here, 
 `captureCodePostal()` will be evaluated only if `cityStartsWith('Paris')`. It might be the case, but the naming is then quite wrong.